tests/unit/db_conftest.py

```diff
@@ -107,6 +107,20 @@ def python_query_data() -> List[List[ColType]]:
     ]
 
 
+@fixture
+def server_side_async_id_callback() -> Response:
+    def do_query(request: Request, **kwargs) -> Response:
+        query_response = {"query_id": "1a3f53d"}
```

Comment:
Nit:
```
def server_side_async_id_callback(server_side_async_id) -> Response:
    ...
        query_response = {"query_id": server_side_async_id }
```
So there's a single point where this string is defined.

Reply:
Done.